src/lib/components/Card/__tests__/Card.test.jsx

```diff
@@ -36,11 +36,11 @@ describe('rendering', () => {
     ...colorPropTest,
     [
       { dense: true },
-      (rootElement) => expect(rootElement).toHaveClass('rootDense'),
+      (rootElement) => expect(rootElement).toHaveClass('isRootDense'),
```

Comment:
Might be replaced with shared test as `dense` is used in multiple props.

Reply:
This is a bit tricky. Dense prop is used in `Card`, `Toolbar` and `ToolbarGroup`. In toolbar related components we use a single SCSS file to re-use code, which means that there is `.isDense` class for both components (@adamkudrna is it okay this way? Or should it be `.isToolbarDense` for `Toolbar` and `.isGroupDense` for `ToolbarGroup`?). In `Card` we use `isRootDense` because this class is used only for the root element. It does make sense to me this way but it makes the `densePropTest` usable either for the toolbar related components only or the `Card` component only.

Reply:
The code of `Toolbar` seems a little bit obsolete in more ways…

1. I'm aware of the inconsistence of `.isDense`, `.isAlignedToTop` etc. Back in those days, the idea was to save some CSS and share everything shareable. However it's inconsistent with the rest of the code base. I think now it's a great chance to fix it. That means being **more explicit in modifier class names**, i.e. have separate class names for individual sub-components:

   ```css
   .isToolbarAlignedToTop,
   .isGroupAlignedToTop {
       align-items: flex-start;
   }
   ```

   or:

   ```css
   :is(.isToolbarDense, .isGroupDense),
   :is(.isToolbarDense, .isGroupDense) .group {
       gap: theme.$gap-dense;
   }
   ```

   or:

   ```css
   :is(.isToolbarNowrap, .isGroupNowrap) > .item:not(.isItemFlexible) {
       flex: 0 1 auto;
   }
   ```
2. According to the latest convention, `alignClass` and `justifyClass` **helper functions** should be placed in the `_helpers` directory, although class name splitting above will make this more wordy.
3. Looking at the modifier classes, I think we could further unify their names instead of attempting to build sentence-like names: `.isToolbarAlignedToTop` ➡️ `.isToolbarAlignTop`, etc. The pattern would be:
   1. `.is<ELEMENT><PROP-NAME><PROP-VALUE>` for `oneOf` prop types, e.g. `.isRootColorPrimary`,
   2. `.is<ELEMENT><PROP-NAME>` for truthy boolean prop types, e.g. `.isRootBlock`.

   If you agree, I'd do it in a separate PR (or a commit at least) and also [document it](https://react-ui.io/contribute/css#class-naming-rules) in part 4.

Reply:
ad 1 & 2: Should be done.
ad 3: Ok, I'm going to create new issue for it.